src/modelgauge/annotation_pipeline.py

```diff
@@ -61,6 +53,13 @@ def key(self, item):
             request = request.model_dump_json()
         return (request, annotator_uid)
 
+    def handle_item(self, item):
+        result = super().handle_item(item)
+        sut_interaction, annotator_uid = item
```

Comment:
I'm not seeing where the deduping is happening (I don't know the pipeline codebase). Can you explain?

Reply:
I think so :)

Reply:
I'm also finding this mysterious. Shouldn't we just add the prompt id to the cache key?